ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+# Skopeo Roadmap
+
+The Skopeo development team reviews feature requests from its various stakeholders for consideration
+quarterly.  Skopeo maintainers then prioritize these features.   Top features are then assigned to
+one or more engineers.
+
+
+## Future feature focus
+
+* OCI artifact support
```

Comment:
Not… that much? `skopeo copy` already handles them.

I do think there is one major gap: “better `skopeo inspect`”, to support both containers and images: https://github.com/containers/skopeo/issues/1283 / https://github.com/containers/skopeo/issues/1933 .

`* A new mode for skopeo inspect to better handle artifacts and multi-platform images`, maybe?